File: controller.cpp
#include <sys/socket.h>
#include <string.h>
#include <netinet/in.h>
#include <arpa/inet.h>
#include <pthread.h>
#include <unistd.h>
#include <stdio.h>

#include "rcp.h"
int main(int argc, char** argv){
    if(argc < 2){
        printf("Usage: bash controller <rover_ip>");
    }
    init();
    connect(argv[1],5555, true);
}

File: controller.py
import rcppy
import sys
import time
import numpy as np

if len(sys.argv) <2:
    print("Usage: python3 controller.py <rover_ip>")
    exit(1)

agent = rcppy.rcppy()
agent.connect(sys.argv[1],5555,False)
agent.listen(1024)
while True:
    agent.send(b'hello',sys.argv[1],5555)
    time.sleep(1)
    pass